jetty-websocket/websocket-core-tests/src/test/java/org/eclipse/jetty/websocket/core/autobahn/AutobahnTests.java

```diff
@@ -88,6 +88,9 @@ public static void before() throws Exception
     @Test
     public void testClient() throws Exception
     {
+        if(!Boolean.getBoolean("autobahn.tests")){
+            return;
```

Comment:
@olamy it would be good if we could still run the tests in intellij without having to set a system property. Is there anyway we can do this.

Reply:
ah yes good point (haha because I'm not running test via IDE :) ) I will change that

Reply:
@lachlan-roberts should work now